ci/gitlab/.gitlab-ci.yml

```diff
@@ -408,3 +408,44 @@ trigger-docsite-build:
   trigger:
     project: olp/edge/ota/documentation/ota-connect-docs
     branch: master
+
+trigger-osx-build:
+  stage: trigger
+  trigger:
+    project: olp/edge/ota/connect/client/homebrew-otaconnect
+    branch: master 
+  rules:
+    - if: $OSX_BUILD
+
+build-osx-release:
+  stage: trigger
+  dependencies:
+    - github-release
+  variables:
+    VERSION: "$CI_COMMIT_TAG"
+    RELEASE_BASE_URL: "https://github.com/advancedtelematic/aktualizr/releases/download"
+    GITHUB_REPOSITORY: "advancedtelematic/aktualizr"
+    GITHUB_TOKEN: "$GITHUB_API_TOKEN"
+  before_script:
+    - brew uninstall -f aktualizr
+    - brew install ghr
+  script:
+    - brew tap advancedtelematic/otaconnect
+    - brew install -v --build-bottle aktualizr
+    - aktualizr --version
+    - brew bottle --json --no-rebuild --force-core-tap --root-url=${RELEASE_BASE_URL}/${VERSION} aktualizr
+    - brew bottle --merge --write --no-commit ./aktualizr--${VERSION}.mojave.bottle.json
+    - brew cat aktualizr | tee aktualizr.rb
+    - mv aktualizr--${VERSION}.mojave.bottle.tar.gz aktualizr-${VERSION}.mojave.bottle.tar.gz
+    - ls -alrt aktualizr-${VERSION}.mojave.bottle.tar.gz
```

Comment:
`ls`?

Reply:
just for testing/debugging purposes - it does not cause any issue while might be helpful for troubleshooting if something goes wrong.